src/main/java/gameboy/gamepack/racinggame/data/entity/Referee.java

```diff
@@ -0,0 +1,49 @@
+package gameboy.gamepack.racinggame.data.entity;
+
+import java.util.*;
+
+import gameboy.gamepack.racinggame.data.vo.CarStatus;
+import gameboy.gamepack.racinggame.data.vo.RaceLog;
+
+public class Referee {
+
+    private RaceRecorder recorder;
+    private static final ThreadLocal<Integer> winnerPosition = ThreadLocal.withInitial( () -> 0);
```

Comment:
```suggestion
    private static final ThreadLocal<Integer> winnerPosition = ThreadLocal.withInitial( () -> 0);
    
    private RaceRecorder recorder;
```

클래스 변수는 멤버변수 위에 배치하는 것이 관례입니다.